test/smoke/kogitoapp.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/util"
+)
+
+const (
+	//kogitoExamplesURI = "https://github.com/kiegroup/kogito-examples"
```

Comment:
Why is this value commented out?

Reply:
I needed to use my own fork of the kogito-examples as the kiegroup quarkus examples are missing the infinispan application.properties for persistence ...

Reply:
@ricardozanini wasn't it supposed to be corrected if properties are missing ?

Reply:
I think this should be fixed in Quarkus 1.1.0?

Reply:
Anyway, please add a comment there, explaining the reasons for substitution, so it can be reverted later.